packages/android_intent/ios/android_intent.podspec

```diff
@@ -3,15 +3,18 @@
 #
 Pod::Spec.new do |s|
   s.name             = 'android_intent'
-  s.version          = '0.3.7'
-  s.summary          = 'A new flutter plugin project.'
+  s.version          = '0.0.1'
```

Comment:
Reverted from https://github.com/flutter/plugins/pull/2598/files#diff-3246420468c6f60fcfdceb079697c3b2R6, this shouldn't be a real version, versioning is handled by pub, not CocoaPods.